subsys/logging/log_backend_rtt.c

```diff
@@ -221,7 +221,10 @@ LOG_OUTPUT_DEFINE(log_output, IS_ENABLED(CONFIG_LOG_BACKEND_RTT_MODE_BLOCK) ?
 static void put(const struct log_backend *const backend,
 		struct log_msg *msg)
 {
-	log_backend_std_put(&log_output, 0, msg);
+	u32_t flag = IS_ENABLED(CONFIG_LOG_BACKEND_RTT_SYST_ENABLE) ?
```

Comment:
it doesn't seem good that we have to patch all the backends like this specifically for one output formatting type. it doesn't seem like this will scale well to additional format types as they are added. is there some other way we can do this in more centralized code?

Reply:
alternative is enabling syst formatting for all backends at once. However, as i'm not familiar with syst i don't know what is the exact use case. How does it match with formatting used for net or bluetooth.

Reply:
currently I don't think there is a centralized way to enable syst formatting for all backends, because besides standard logger backends, there are non-standard backends like net backend or bluetooth backend.

Reply:
@nordic-krch @andrewboie @nashif besides centralized issue, is there any other concerns for this PR? Thanks